test/screenshot/config.js

```diff
@@ -44,7 +44,10 @@ for (const fundingSource of SUPPORTED_FUNDING_SOURCES) {
     buttonConfigs.push({
         button: {
             fundingSource,
-            ...DEFAULT_OAUTH_PROPS
+            ...DEFAULT_OAUTH_PROPS,
+            style: {
+                ...(fundingSource === FUNDING.CREDIT ? { color: BUTTON_COLOR.DARKBLUE } : {})
```

Comment:
We should default to darkblue if the funding source is credit, without the merchant needing to pass it explicitly.